parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -340,6 +344,19 @@ ensureTeleportd() {
     systemctlEnableAndStart teleportd || exit $ERR_SYSTEMCTL_START_FAIL
 }
 
+ensureArtifactStreaming() {
+  systemctl enable acr-mirror.service
+  systemctl start acr-mirror.service
+  sudo /opt/acr/tools/overlaybd/install.sh || exit ERR_ARTIFACT_STREAMING_INSTALL
```

Comment:
these codes are gonna get masked anywhere by the top-level, do we really need them here?

Reply:
reverted to my original script that didn't have it